stack.yaml

```diff
@@ -82,12 +82,9 @@ extra-deps:
     # dependencies of iohk-monitoring
   - prometheus-2.1.2
   - libsystemd-journal-1.4.4
+  - katip-0.8.3.0
   - git: https://github.com/CodiePP/ekg-prometheus-adapter
     commit: 1a258b6df7d9807d4c4ff3e99722223d31a2c320
-  - git: https://github.com/Soostone/katip.git
-    commit: cf60f0e14640ae9702c111bb51aeca7eed50ad48
-    subdirs:
-      - katip
```

Comment:
Wait, why are you undoing this change? This commit includes a PR that @coot made (https://github.com/Soostone/katip/commit/cf60f0e14640ae9702c111bb51aeca7eed50ad48) which we need for making `ouroboros-network` work on Windows.

Reply:
yes, but I don't see it referenced in master of ouroboros-network (cabal.project, stack.yaml)

Reply:
@coot confirmed that we can stick to katip 0.8.3.0